drivers/nrf_802154_sl_opensource/src/platform/hp_timer/nrf_802154_hp_timer.c

```diff
@@ -88,8 +88,6 @@ void nrf_802154_hp_timer_deinit(void)
 
 void nrf_802154_hp_timer_start(void)
 {
-    // This resource is shared, so its configuration must be restored before every use
-    // instead of setting in initialization only.
```

Comment:
Is this comment removed intentionally?

Reply:
Definitely not, it's a mistake. Thanks for catching it!